packages/geoprocessing/storybook.tsx

```diff
@@ -37,11 +38,12 @@ export const registerExampleStories = (
         sketchProperties: {
           name: "My Sketch",
           updatedAt: new Date().toISOString(),
-          sketchClassId: "abc123"
+          sketchClassId: "abc123",
         } as SketchProperties,
         projectUrl: "https://example.com/project",
         exampleOutputs: [],
-        simulateLoading: true
+        simulateLoading: true,
+        visibleLayers: [],
```

Comment:
Is this required by design?  Like you want an implementer to be very clear the report has no visible layers?  Seems like it could be optional or just have a default value of empty perhaps

Reply:
I was lazy here. Ideally there would be a function to allow simulating the toggle of layers. With this set to an empty array clicking on the switch in storybook will do nothing. I'd have to restructure the way the component state is setup here and just ran out of time. I should probably add that before merging.